tests/TestEndpoints.js

```diff
@@ -26,8 +26,7 @@ export class TestEndpoints {
   async issue(credential) {
     const {issuer} = this;
     const issueBody = createRequestBody({issuer, vc: credential});
-    const response = await post(issuer, issueBody);
-    return response?.verifiableCredential || response;
```

Comment:
Curious what caused this change. Does this never happen now?

Reply:
@aljones15 pointed out [this section to me](https://github.com/w3c/vc-test-suite-implementations/blob/056b8d6e00275ce13656a372342c770125ef9264/lib/requests.js#L115-L124). I tested it again to confirm and I can't reproduce the behavior I had before

Reply:
So, I guess the "format is stabilized" enough to remove it then. Hooray for simplicity. 😄